frontend/src/scenes/billing/AllProductsPlanComparison.tsx

```diff
@@ -331,6 +331,7 @@ export const AllProductsPlanComparison = ({
             <h3>Product features breakdown:</h3>
             <LemonCollapse
                 defaultActiveKey={product.type}
+                size="large"
```

Comment:
The font size on the large LemonCollapse feels awkwardly large, so I would vote for keeping the regular size here. Before vs. after:
<img width="1426" alt="Screenshot 2025-01-03 at 16 19 16" src="https://github.com/user-attachments/assets/106e8c40-9aac-4c2e-932f-9428d1ee4d4a" />

Reply:
Yeah this was a middle ground. I would need to override with some custom CSS instead tbh